### ISSUE TITLE
[Accessibility] Unable to navigate between action buttons using `Tab` in confirmation dialog

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#28891]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.81.1
- OS Version: macOS Ventura 13.4.1

Steps to Reproduce:

1. Open a new editor tab with `Cmd` + `N`
2. Type some text
3. Close the tab using `Cmd` + `W`; a confirmation dialog appears, reminding you to save before closing
4. Navigate between the "Save," "Don't Save," and "Cancel" buttons using the `Tab` key

Hi VSCode team,

Thanks for all your hard work. 

I came across this related to accessibility that I couldn't find in the known -. As mentioned above, the action buttons in the confirmation dialog should be tappable.

<img width="479" alt="截圖 2023-08-14 上午12 46 45" src="https://github-com/microsoft/vscode/assets/26245542/fae2d638-37b1-4b2f-b287-23b70f65e42c">

I'd love to help with this if it's ok. Thanks!
